src/lasdi/param.py

```diff
@@ -78,6 +118,40 @@ def createTestGridSpace(self, param_list):
         mesh_grids = self.createHyperMeshGrid(paramRanges)
         return gridSizes, mesh_grids, self.createHyperGridSpace(mesh_grids)
     
+    def createTestGridSpaceForHull(self, param_list):
+        '''
+        This is similar to createTestGridSpace, but with some different variables.
+        We take the min/max value of each parameter, and create a uniform rectangular grid
+        over the parameter space with 'sample_size' points in each dimension. 
+        '''
+
+        paramRanges = []
+        gridSizes = []
+
+        for param in param_list:
+            Nx, _ = getParam1DSpace[param['test_space_type']](param)
+            minval = param['min']
+            maxval = param['max']
+            gridSizes += [Nx]
+            paramRanges += [np.linspace(minval, maxval, Nx)]
```

Comment:
I think we can simply use `uniform` param 1d space here, instead of re-writing it. Line 132-136 could be:
```
Nx, paramRange = getParam1DSpace[param['test_space_type']](param)
gridSizes += [Nx]
paramRanges += [paramRange]
```

If the param space cannot be in log scale, I suggest adding a `assert` to enforce it.

Reply:
Also followed this suggestion to avoid introducing the `exterior` test_space_type